### ISSUE TITLE
script-src with multiple values is incorrectly detected as insecure

### ISSUE THREAD

**Issue by Rob--W** [mozilla/addons-linter#4576]
### Describe the problem and steps to reproduce it:

Test case, create `manifest.json` and lint the directory containing the following:
```json
{
  "name": "x",
  "version": "1",
  "manifest_version": 2,
  "content_security_policy": "default-src http:; script-src 'self' 'self'"
}
```

### What happened?

The MANIFEST_CSP warning was emitted.

### What did you expect to happen?

The specified `script-src` directive is secure, so there should not be any linter warnings.

### Anything else we should know?

This bug is caused by the check at https://github.com/mozilla/addons-linter/blob/a2fbda70d90106900d05be18410285afe22d13b9/src/parsers/manifestjson.js#L1059-L1066

It only allows the `script-src` directive to override an insecure `default-src` directive if it consists of just `'self'`.

**Comment by AlexandraMoga** [comment]
I've verified this issue with the latest linter release (5.22.0) and `script-src` with multiple values is no longer raising validation warnings.

[before fix]
![image](https://user-images.githubusercontent.com/31961530/202155440-8917ba90-8af7-4fe8-86e0-004179fab50a.png)


[after-fix]
![image](https://user-images.githubusercontent.com/31961530/202155501-c4b8c1fd-ba1e-42f7-af0a-3994bc16dd89.png)
